docs/automation/terraformer_qs.md

```diff
@@ -0,0 +1,110 @@
+---
+id: terraformer_qs
+title: Terraformer Container
+sidebar_label: Terraformer Container
+---
+
+
+Terraformer Container
```

Comment:
You can add `hide_title: true` to the frontmatter, or can remove this line to get rid of the duplicate title

I would also add `hide_table_of_contents: true` to allow for more room for the code blocks

Reply:
done!